Wikipedia/View Controllers/WebView/NativeSectionTitleOverlays/WMFTitleOverlayModel.h

```diff
@@ -0,0 +1,17 @@
+//  Created by Monte Hurd on 7/30/15.
+//  Copyright (c) 2015 Wikimedia Foundation. Provided under MIT-style license; please copy and modify!
+
+#import <Foundation/Foundation.h>
+
+@class WMFTitleOverlayLabel;
+@interface WMFTitleOverlayModel : NSObject
```

Comment:
I know when I mentioned making this class I called it a "Model" class… but it may be helpful to think of it more of a controller class.

It should be "in charge of" the section header that it is assigned. So things like title, label, and constraint should be internal so only this object can modify them. That will make it much easier to debug later on since only one object can make changes.